finale/src/styles/App 2.scss

```diff
@@ -0,0 +1,37 @@
+$backgroundColor: #585a6f;
```

Comment:
Please remove all files appended with `filename 2.xx`, it's unaccepted

Reply:
Removed. Thank you for mentioning this.